Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const inHours = tasks.map(hour => hour['duration']/60);
```

Comment:
What you map here are tasks. The callback function that you pass to .`map()` receives one task at a time. Therefore it is logical to name the parameter of that function `task` and not `hour`.

You should use dot-notation to access properties from object if not know the name of the property beforehand, which is true in the case. The result of the `.map()` method in this case is an array of durations. Will all that, for clarity, you could rewrite this statement as:

```js
const durations = tasks.map(task => task.duration / 60);
```